Apps/DBMaintainer/Parsers/FederalDrugProductParser.cs

```diff
@@ -110,7 +109,7 @@ public IList<Status> ParseStatusFile(string filePath, IEnumerable<DrugProduct> d
             using CsvReader csv = new(reader, csvConfig);
             TypeConverterOptions options = new()
             {
-                Formats = new[] { "dd-MMM-yyyy" },
+                Formats = ["dd-MMM-yyyy"],
                 DateTimeStyle = DateTimeStyles.AssumeUniversal | DateTimeStyles.AdjustToUniversal,
```

Comment:
Should this have been deleted like line 59?

Reply:
No, because this method still deals with DateTimes.  The change on line 112 is just code formatting, while the logic remains as is.